src/Common/ISymbolExtensions.cs

```diff
@@ -210,7 +210,7 @@ internal static bool IsMoqCallbackMethod(this ISymbol symbol, MoqKnownSymbols kn
     /// <returns>True if the symbol is a Raises or RaisesAsync method from Moq.Language interfaces; otherwise false.</returns>
     internal static bool IsMoqRaisesMethod(this ISymbol symbol, MoqKnownSymbols knownSymbols)
```

Comment:
nit: I think we can delete this method in favor of `IsKnownMoqRaisesMethod` now.

Reply:
(and likely remove the "known" part of the name now)